pom.xml

```diff
@@ -14,7 +14,7 @@
     <groupId>org.mule.connectors</groupId>
     <artifactId>mule-sftp-connector</artifactId>
     <packaging>mule-extension</packaging>
-    <version>2.3.0-SNAPSHOT</version>
+    <version>2.3.0-Perf-Test-SNAPSHOT</version>
```

Comment:
We need to change this rt?

Reply:
Made this change for performance testing. Reverted it